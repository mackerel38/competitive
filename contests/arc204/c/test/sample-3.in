9
1 6 7 3 5 8 9 2 4
10
5 3 1
0 7 2
0 6 3
1 3 5
4 0 5
4 2 3
1 5 3
4 5 0
2 2 5
2 5 2
