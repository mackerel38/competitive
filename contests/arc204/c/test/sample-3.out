14
0
0
4
7
11
4
13
8
8
